crates/matrix-sdk/src/sliding_sync/room.rs

```diff
@@ -153,14 +153,33 @@ impl SlidingSyncRoom {
         } = room_data;
 
         self.inner.unread_notifications = unread_notifications;
```

Comment:
Why don't we need to treat this the same way as the others?

Reply:
It wasn't the case before, so I guess no… The specification doesn't mention anything specifically.